src/app/api/application/uniform/route.ts

```diff
@@ -157,4 +166,14 @@ export async function PUT(req: NextRequest) {
       },
     );
   }
+
+  function checkAllowedUpdateFields(
```

Comment:
duplicate code, please abstract and use same function in both routes

Reply:
Fix it!